src/main/kotlin/pt/up/fe/ni/website/backend/model/Activity.kt

```diff
@@ -30,6 +30,10 @@ abstract class Activity(
     @OneToMany(fetch = FetchType.EAGER)
     open val teamMembers: MutableList<Account>,
 
+    @JoinColumn // Fixme: Don't know if this is needed
+    @OneToMany(fetch = FetchType.EAGER)
+    open val hallOfFame: MutableList<Account>,
```

Comment:
This shouldn't be in the activity, since the hall of fame is a feature for projects only (correct if I'm wrong)